templates/blog/macros.html

```diff
@@ -2,7 +2,7 @@
   <ul>
   {% for article in articles %}
   {% set url = url_for_article(article) %}
-  <li class=articles>
+  <li class=article>
```

Comment:
I still wonder why you need a `ul > li` (with a class article!) to have an… article?
We did things like that when the `<article>` tag didn't exist